### ISSUE TITLE
cannot build with xcodebuild

### ISSUE THREAD

**Issue by scottiedog45** [andyfinnell/NativeMarkKit#3]
First let me say I really like this repo- it's super handy! Thanks for the work. 

Somehow I'm unable to build a project via `xcodebuild` with this package. 

Steps to repro:

- New xcode proj
- Add spm nativemarkkit
- from command line: `xcodebuild -scheme 'scheme' -sdk iphoneos`

I get the error: 
```
The following build commands failed:
CompileSwift normal armv7 /Users/me/Library/Developer/Xcode/DerivedData/marktest-epvvpnjxgaeusqfjffrlspcvjsnw/SourcePackages/checkouts/NativeMarkKit/Sources/NativeMarkKit/render/NativeMarkText+UIKit.swift
CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler
```

Do you have advice on resolving this? 

Thanks very much 💯 

Using `Version 12.3 (12C33)`

**Comment by pgorrindo** [comment]
@andyfinnell hope your 2021 is off to as good a start as possible! I'm having this same issue -- any thoughts? Love this project, thanks for making it available!

**Comment by andyfinnell** [comment]
Thanks for writing this up. I can confirm I can reproduce this. I'll work on a fix tonight.

**Comment by andyfinnell** [comment]
It appears this is happening because the containing app is building the armv7 architecture, and SwiftUI doesn't support that architecture. I can see in my build logs it builds fine for armv64 though.

According to these [two](https://forums.swift.org/t/package-swiftui-ios-10/30483) [posts](https://stackoverflow.com/questions/61954607/swiftui-use-of-undeclared-type-xxx-on-deployment-targets-below-or-equal-to-i) it can happen when the deployment target is iOS 10 or less. However, I'm seeing it in a test app that targets iOS 13.5. So I'm a bit baffled as to why it's trying to build armv7 at all.

**Comment by andyfinnell** [comment]
I think I've fixed it in v1.1.1. Let me know!

**Comment by pgorrindo** [comment]
It's working for me now -- woohoo! Thanks!
